.github/workflows/nodejs.yaml

```diff
@@ -18,4 +18,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - run: npm test
+      - run: npm run predeploy
+      - run: aws s3 cp lambda.zip s3://covid-vaccine-scraper-test/lambda.zip
```

Comment:
@j256 Copying data into S3 will require credentials that are used by the hosted GitHub runner for the workflow. In addition, S3 bucket names are globally unique so I would recommend parameterizing the bucket destination so that the repository owner can provide the value through a GitHub secret.

Reply:
@kriation good call on S3 creds and making the bucket a param, should be possible via GitHub .
@j256 As discussed yesterday we should probably trigger a different workflow for deploying to lambda on PR merge to master (or develop)